A3A/addons/core/functions/Missions/fn_RES_Deserters.sqf

```diff
@@ -0,0 +1,274 @@
+}; 
+
+private _nearbyPos = [_spawnPos, 200, 300, 3, 0, 5, 0] call BIS_fnc_findSafePos;
+
+private _patrolGroup1 = [_nearbyPos, _sideX, _infantrySquadArray] call A3A_fnc_spawnGroup;
+{ 
+    [_x] call A3A_fnc_NATOinit;
+} forEach units _patrolGroup1;
+
+private _PatrolvehData  = [_nearbyPos, 0,_vehiclePatrolType, _sideX] call A3A_fnc_spawnVehicle;
+private _Patrolveh = _PatrolvehData select 0;
+private _vehCrew = _PatrolvehData select 1;
+private _patrolVehgroup = _PatrolvehData select 2;
+{ 
+    [_x] call A3A_fnc_NATOinit;
+} forEach _vehCrew;
+[_Patrolveh, _sideX] call A3A_fnc_AIVEHinit;
+
+private _stolenVehicleSpawnPos = [_spawnPos, 5, 30, 3, 0, 5, 0] call BIS_fnc_findSafePos;
+private _stolenVehicle = createVehicle [_stolenVehicleType, _stolenVehicleSpawnPos, [], 0, "NONE"];
+[_stolenVehicle, teamPlayer] call A3A_fnc_AIVEHinit;
+//private _stolenVehgroup = group driver _StolenVehGroup;///not sure how to transfer command of units inside vehicle to player, so for now vehicle will stay empty
+
+private _patrolGroup2 = [];
+private _soldersPatrol = [];
+if (_difficultX) then {
+	_nearbyPos = [_spawnPos, 100, 150, 3, 0, 20, 0] call BIS_fnc_findSafePos;
+	_patrolGroup2 = [_nearbyPos, _sideX, _infantrySquadArray] call A3A_fnc_spawnGroup;
+	_soldersPatrol append units _patrolGroup2;
+	{ 
+    	[_x] call A3A_fnc_NATOinit;
+	} forEach units _patrolGroup2;
+};
+_soldersPatrol append units _patrolGroup1;
+_soldersPatrol append units _patrolVehgroup;
+waitUntil {
+    sleep 1;
+    (call SCRT_fnc_misc_getRebelPlayers) inAreaArray [_spawnPos, 500, 500] isNotEqualTo [] || {dateToNumber date > _dateLimitNum}
+};
+
+[_patrolGroup1, _stolenVehicleSpawnPos, 15] call bis_fnc_taskPatrol;
+[_patrolVehgroup, _stolenVehicleSpawnPos, 15] call bis_fnc_taskPatrol;
+if (_difficultX) then {
+	[_patrolGroup2, _stolenVehicleSpawnPos, 15] call bis_fnc_taskPatrol;
+};
+
+private _grpDeserters = createGroup teamPlayer;
+private _unit = objNull;
+private _unitTypes = [(_faction get "unitMilitiaGrunt"),(_faction get "unitMilitiaMarksman"),
+(_faction get "unitMilitiaGrenadier"),(_faction get "unitMilitiaSniper"),
+(_faction get "unitMilitiaMedic"),(_faction get "unitCrew"),(_faction get "unitPilot"),
+"loadouts_occ_militia_Grenadier","loadouts_occ_military_Grenadier","loadouts_occ_elite_Grenadier",
+"loadouts_occ_militia_LAT","loadouts_occ_military_LAT","loadouts_occ_elite_LAT",
+"loadouts_occ_militia_MachineGunner","loadouts_occ_military_MachineGunner","loadouts_occ_elite_MachineGunner","loadouts_occ_militia_Rifleman","loadouts_occ_military_Rifleman",
+"loadouts_occ_elite_Rifleman","loadouts_occ_militia_Marksman","loadouts_occ_military_Marksman","loadouts_occ_elite_Marksman","loadouts_occ_militia_Sniper",
+"loadouts_occ_military_Sniper","loadouts_occ_elite_Sniper"];
```

Comment:
Is there any real reason to be listing each individual loadout?

Reply:
I believe so, cause there is no other way of getting enemy loadouts iirc